hook/lua/aibrain.lua

```diff
@@ -3206,243 +3207,13 @@ AIBrain = Class(RNGAIBrainClass) {
     end,
 
     TacticalThreatAnalysisRNG = function(self, ALLBPS)
```

Comment:
This debug code should be removed, it should be: `TacticalThreatAnalysisRNG =import("/mods/RNGAI/lua/aibrain-threatanalysis.lua").TacticalThreatAnalysisRNG(self, ALLBPS)`